### DISCUSSION TITLE
Permission issue?

### DISCUSSION THREAD

**Discussion by tyriekfv** [RandomNinjaAtk/docker-lidarr-extended#200] [General]
Not sure because there isn't anything in that folder. The script pulls from lidarr missing, but not the cutoff.

ls: cannot access '/config/extended/cache/lidarr/list/*-cutoff': No such file or directory

**Comment by RandomNinjaAtk** [comment]
You sure you have anything in your cutoff list?

You would get this error, if you didn't...

**Reply by tyriekfv** [reply]
Yeah it's full

**Reply by RandomNinjaAtk** [reply]
Please post relevant log data, and script version.

**Comment by tyriekfv** [comment]
[Audio.txt](https://github.com/RandomNinjaAtk/docker-lidarr-extended/files/10610133/Audio.txt)
(1.0.307)

it's searched the cutoff list before

**Reply by RandomNinjaAtk** [reply]
Your a couple of versions behind, update to the latest, by pulling a new image and see if that resolves your issue.

**Comment by catduckgnaf** [comment]
Latest Audio.txt 1.0.309. Also, does that make sense for the cutoff directory? should it be?
[audio.txt](https://github.com/RandomNinjaAtk/docker-lidarr-extended/files/11242235/audio.txt)
/config/lidarr-extended/extended/logs/searched/lidarr/artist/
/config/extended/cache/lidarr/list/*-cutoff'

It goes on repeat.
